### PR TITLE
Update product IDS

### PR DESCRIPTION
**Pull request by jigar-f**
Added support for 1 Month plans for Google in-app purchase

### CODE REVIEW

**Comment by jigar-f** [comment]
@atavism I added a new product-Id in the play console, It still not reflecting on the app, Please check after a few hours if working go ahead and test all payment flow, if all working we need to put out a new release for this.

File: android/app/src/main/kotlin/io/lantern/model/SessionModel.kt [hunk 1]
```diff
@@ -161,11 +161,23 @@ class SessionModel(
...
+            "submitGooglePlayPayment" -> {
+                try {
+                    paymentsUtil.submitGooglePlayPayment(
+                        call.argument("email")!!,
+                        call.argument("planID")!!,
+                        result,
+                    )
+                }
+                catch (e: Exception) {
+                    result.error(
```
Comment (atavism) [inline android/app/src/main/kotlin/io/lantern/model/SessionModel.kt:173]: Do you want to log the exception thrown here?

**Review by atavism — APPROVED** [review]
LGTM! Just one comment

File: android/app/src/main/kotlin/io/lantern/model/SessionModel.kt (hunk 1 above)
Comment (jigar-f) [reply android/app/src/main/kotlin/io/lantern/model/SessionModel.kt:173]: We do, so if something goes wrong, it should send the error to Flutter and update the UI according. Before it was just loading forever

**Comment by jigar-f** [comment]
Let's wait till new changes reflect, then we can merge,

File: android/app/src/main/kotlin/io/lantern/model/SessionModel.kt (hunk 1 above)
Comment (atavism) [reply android/app/src/main/kotlin/io/lantern/model/SessionModel.kt:173]: I thought the error being shown is R.string.error_making_purchase and not the actual message from the exception

Comment (jigar-f) [reply android/app/src/main/kotlin/io/lantern/model/SessionModel.kt:174]: @atavism Added log exception.

**Comment by jigar-f** [comment]
@atavism I can see a new plan on google play. I think we can go ahead and merge this.

**Comment by atavism** [comment]
@jigar-f Sounds great. Merging!

**Pull request merged**